akka-remote/src/main/resources/reference.conf

```diff
@@ -98,6 +98,11 @@ akka {
 
     # Acknowledgment timeout of management commands sent to the transport stack.
     command-ack-timeout = 30 s
+    
+    # The timeout for outbound associations to perform the handshake.
+    # If the transport is akka.remote.netty.tcp or akka.remote.netty.ssl
+    # the configured connection-timeout for the transport will be used instead.
```

Comment:
I am not sure about that. The connection-timeout relates to TCP events, which is handled by the kernel, while the Akka Protocol is at the JVM level. I am not sure these timeouts mix well.